gno.land/pkg/sdk/vm/convert_json.go

```diff
@@ -0,0 +1,38 @@
+package vm
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"github.com/gnolang/gno/gnovm/pkg/gnolang"
+	gno "github.com/gnolang/gno/gnovm/pkg/gnolang"
+	"github.com/gnolang/gno/tm2/pkg/amino"
+)
+
+func UnmarshalJSON(alloc *gno.Allocator, store gno.Store, b []byte, t gno.Type) (tv gno.TypedValue, err error) {
+	tv.T = t
+	gvalue := gnolang.Gno2GoValue(&tv, reflect.Value{})
+	v := reflect.New(gvalue.Type())
+	if err := amino.UnmarshalJSON(b, v.Interface()); err != nil {
```

Comment:
verify now or add a comment+issue to verify if we can't be DDOS here.

Reply:
I will create the issue when this will be merge, I will keep this unresolve until then.

Reply:
https://github.com/gnolang/gno/issues/1859